packages/devtools_app/lib/src/screens/logging/logging_controller.dart

```diff
@@ -330,21 +331,21 @@ class LoggingController extends DisposableController
   void _handleDeveloperLogEvent(Event e) {
     final VmServiceWrapper? service = serviceConnection.serviceManager.service;
 
-    final logRecord = e.json!['logRecord'];
+    final logRecord = _LogRecord(e.json!['logRecord']);
 
     String? loggerName =
-        _valueAsString(InstanceRef.parse(logRecord['loggerName']));
+        _valueAsString(InstanceRef.parse(logRecord.loggerName));
     if (loggerName == null || loggerName.isEmpty) {
       loggerName = 'log';
     }
-    final int? level = logRecord['level'];
-    final InstanceRef messageRef = InstanceRef.parse(logRecord['message'])!;
+    final int? level = logRecord.level;
+    final InstanceRef messageRef = InstanceRef.parse(logRecord.message)!;
     String? summary = _valueAsString(messageRef);
     if (messageRef.valueAsStringIsTruncated == true) {
       summary = '${summary!}...';
     }
-    final InstanceRef? error = InstanceRef.parse(logRecord['error']);
-    final InstanceRef? stackTrace = InstanceRef.parse(logRecord['stackTrace']);
+    final InstanceRef? error = InstanceRef.parse(logRecord.error);
+    final InstanceRef? stackTrace = InstanceRef.parse(logRecord.stackTrace);
```

Comment:
nit all of these can have types removed on the LHS now (matches devtools style and now these are strongly typed)

Reply:
Sweet